iceoryx_posh/include/iceoryx_posh/internal/roudi/service_registry.hpp

```diff
@@ -46,8 +46,13 @@ class ServiceRegistry
     {
         ServiceDescriptionEntry(const capro::ServiceDescription& serviceDescription);
 
-        capro::ServiceDescription serviceDescription{};
-        ReferenceCounter_t count{1U};
+        capro::ServiceDescription serviceDescription;
+
+        // it can be discussed whether the type of service (server or publisher)
```

Comment:
Should this comment not be in some issue instead of in the code?

Reply:
I wanted to ad the rationale behind not encoding the information into the ServiceDescription (discussed it with @elBoberido ) and deemed it relevant in the code.

There is also some uncertainty about details how to handle the publisher/server distinction and there are differences that are relevant for the registry (e.g. there can only be one server of a specific description but it can be the same description as some publisher etc.).

Can be removed though, i am indifferent here.